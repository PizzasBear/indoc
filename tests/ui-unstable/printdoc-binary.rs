#![feature(proc_macro_hygiene)]

use indoc::printdoc;

fn main() {
    printdoc!(b"");
}
