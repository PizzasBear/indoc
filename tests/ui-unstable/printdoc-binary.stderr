error: byte strings are not supported in formatting macros
 --> $DIR/printdoc-binary.rs:6:15
  |
6 |     printdoc!(b"");
  |               ^^^
